lib/validators/options_processor/options/must_exclude.js

```diff
@@ -34,9 +35,23 @@ class MustExclude {
         return !regexObj.test(input)
       } else if (Array.isArray(input)) {
         if (filter.all) {
-          return input.every(label => !regexObj.test(label))
+          return input.every(label => {
+            if (typeof label === 'string') {
```

Comment:
Not a big deal but I noticed this part of the code is abstracted out to `arrayHandler` in `must_include.js`, is there a reason it is not done here?

Reply:
mainly because in the must_exclude case, the logic in filter.all and the else statement are slightly different:

`input.every(label => !regexObj.test(label))`

vs

`!input.some(label => regexObj.test(label))`

notice the position of the `!`